diff --git a/lib/cjs/inject.js b/lib/cjs/inject.js
index 3c23f24c4a686b7b73f7ef16858e6fc91ed6d9e7..33aa188ad5eb36fc777502e8938b4fa6f1c936ff 100644
--- a/lib/cjs/inject.js
+++ b/lib/cjs/inject.js
@@ -181,7 +181,7 @@ function getOrderedGroups(obj) {
   return Object.keys(obj).map(Number).sort((a, b) => a > b ? 1 : -1);
 }
 const selectorPattern = /\s*([,])\s*/g;
-const conditionalRulePattern = /^@(media|supports|container)\s*\([^)]+\)\s*{/;
+const conditionalRulePattern = /^@(media|supports|container)\b/;
 function getSeenRuleKey(cssText) {
   if (conditionalRulePattern.test(cssText)) {
     const index = cssText.indexOf('{');
diff --git a/lib/es/inject.mjs b/lib/es/inject.mjs
index 446b0cc616208c5df97739ca21c96238d05cb507..4124c1b37aa599bbb2cb02b566465b10566b05a3 100644
--- a/lib/es/inject.mjs
+++ b/lib/es/inject.mjs
@@ -179,7 +179,7 @@ function getOrderedGroups(obj) {
   return Object.keys(obj).map(Number).sort((a, b) => a > b ? 1 : -1);
 }
 const selectorPattern = /\s*([,])\s*/g;
-const conditionalRulePattern = /^@(media|supports|container)\s*\([^)]+\)\s*{/;
+const conditionalRulePattern = /^@(media|supports|container)\b/;
 function getSeenRuleKey(cssText) {
   if (conditionalRulePattern.test(cssText)) {
     const index = cssText.indexOf('{');
